### PR TITLE
Web Scraping Tools

### PR DESCRIPTION
**Pull request by OwenCochell**
I have altered the video download script to retrieve vidoes from https://www.signasl.org. Some more work needs to be done, but for now this should work for now.

The script will navigate to the search results given to it, and will download all videos on the page. It also sorts the videos by name, as some search results have videos of diffrent names.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-tdqm==0.0.1
-flask
\ No newline at end of file
+opencv-contrib-python
+opencv-python
+pandas
+pytube
+tdqm
+flask
+yt-dlp
```
Comment (nomppy) [inline requirements.txt]: bs4, requests are required but not in requirements.txt

**Review by nomppy — CHANGES_REQUESTED** [review]
I think you mean `asl_down`
Also, a couple libraries are missing from requirements.txt

File: requirements.txt (hunk 1 above)
Comment (OwenCochell) [reply requirements.txt]: Added in new commits

**Comment by OwenCochell** [comment]
All issues should be corrected in the latest commit, let me know if their are any more issues!

**Review by nomppy — APPROVED** [review]
LGTM

**Pull request merged**